usagov/usagov-2021
# USAGOV-1909-breadcrumb-scoping: Create new system for naming and scoping wizard breadcrumbs

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1909

## Description

Create new wizard breadcrumb field. Send information from it to wizard twig.

## Type of Changes

- [ X ] New Feature
- [ X ] Frontend (Twig, Sass, JS)
<img width="821" alt="image" src="https://github.com/user-attachments/assets/4fc35167-1981-4af9-bbe4-3f5efd6225ab">
- [ X ] Drupal Config (requires "drush cim")

## Testing Instructions


1. Go to any wizard page other than the root.
2. The page loads as expected.
3. Edit the page. 
4. The edit screen loads as expected.
5. Go to the root term of a wizard tree and edit the page.
6. Scroll down to the new "Breadcrumbs" field, and add a link to the "Scams and fraud" page, as well as a `<nolink>` to "Where to report a scam". Save.
7. Navigate through the wizard. 
8. The breadcrumbs match the breadcrumbs shown in the screenshot.
9. Repeat the process on a Spanish page.
10. You get similar results, this time in Spanish.
11. 
### Change Requirements



## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
12. Find the commit of this pull request.
13. Build and deploy the changes.
14. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
